.github/workflows/import_resources_to_tf.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: Import TF Resources
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy_management:
+    name: Import TF Resources
+    runs-on: ubuntu-latest
+    environment: Dev
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Import Resources
+        uses: ./.github/actions/devcontainer_run_command
```

Comment:
why is it using a devcontainer script?
If it is not related to devcontainer maybe this code should be extracted?